people/akasa_musha/page.en.md

```diff
@@ -8,11 +8,11 @@ Yuxue is a person who craves companionship, which was often mentioned in her twe
 
 She had the side of being a little girl. She used to decorate mangoes with candles. She also did some research on electronics.
 
-Her parents expressed serious disapproval of her transgender behavior at first, but eventually gained a little understanding of her after her efforts. Like other Chinese schools, her school prevented her from carrying HRT medicines to school and keeping her hair long, which may have exacerbated her depression.
+Her parents expressed serious disapproval of her transgender behaviour at first, but eventually gained a little understanding of her after her efforts. Like other Chinese schools, her school prevented her from carrying HRT medicines to school and keeping her hair long, which may have exacerbated her depression.
 
 It is a pity that her friends’ efforts did not stop her depression from worsening.
 In the end, she chose suicide.
 
-After her death, Han Lianyi (a Chinese trans women known for her volunteer suicide prevention work) reflected on her failure to transfer Yuxue to someone else in time when Han was not capable enough for her case; failing to discover Yuxue's obvious signs of suicide in time; and her inadequacy in guiding the rescue of Yuxue at hospital. Han hopes that this failure could help guide future suicide prevention efforts.
+After her death, Han Lianyi (a Chinese transwoman known for her volunteer suicide prevention work) reflected on her failure to transfer Yuxue to someone else in time when Han was not capable enough for her case; failing to discover Yuxue's obvious signs of suicide in time; and her inadequacy in guiding the rescue of Yuxue at hospital. Han hopes that this failure could help guide future suicide prevention efforts.
```

Comment:
transwoman is not a real word. It's also often used as a degrading term to say trans women, somehow, are fundamentally not "women".

trans woman, however, is adj + noun, means a woman who's trans.

Should be "trans woman".

Reply:
Apologies for my oversight, I was not aware.